code/modules/research/techweb/nodes/tools_nodes.dm

```diff
@@ -13,15 +13,15 @@
 	display_name = "Mining Technology"
 	description = "Better than Efficiency V."
 	prereq_ids = list("engineering", "basic_plasma")
-	design_ids = list("drill", "superresonator", "triggermod", "damagemod", "cooldownmod", "rangemod", "ore_redemption", "mining_equipment_vendor", "cargoexpress")//CB Edit Yeet Plasma cutter
+	design_ids = list("drill", "superresonator", "triggermod", "damagemod", "cooldownmod", "rangemod", "ore_redemption", "mining_equipment_vendor", "cargoexpress", "plasmacutter")//e a r l y    g a  m e)
```

Comment:
Unsure if the plasma cutter was nintended to be readded? but eh, seems good, idk what implications it has.

Reply:
Then it's guuchi?